modules/samples-operator-configuration.adoc

```diff
@@ -114,7 +114,7 @@ imagestreams is in the reason field.
 previously are submitted.
 
 |`RemovePending`
-|Indicator that there is a `Management State: Removed` setting pending, but are
+|Indicator that there is a `Management State: Removed` setting pending, but is
```

Comment:
I'm not sure what is waiting. The samples Operator itself? Will you add a noun, too?